progenitor-impl/src/lib.rs

```diff
@@ -400,7 +400,8 @@ impl Generator {
             "percent-encoding = \"2.1\"",
             "reqwest = { version = \"0.11\", features = [\"json\", \"stream\"] }",
             "serde = { version = \"1.0\", features = [\"derive\"] }",
-            "serde_urlencoded = 0.7",
+            "serde_json = \"1.0\"",
+            "serde_urlencoded = \"0.7\"",
```

Comment:
form_urlencoded was supposed to use serde_urlencoded not serde_json

Reply:
https://github.com/oxidecomputer/progenitor/blob/0c7cace799bde609c87d1aeb9b093bc12d14f180/progenitor-client/src/progenitor_client.rs#L338

here?

Reply:
Yes. I'm happy to sort that out if you'd prefer; it was an oversight on my part